### ISSUE TITLE
Get a link from a couple (username, socialLink)

### ISSUE THREAD

**Issue by bacloud23** [gkucmierz/social-links#41]
Hi @gkucmierz 
Thanks a lot for providing this handy library. 
Is there a way to reverse ? from a username and an identified social media site give the correct mobile and absolute link

Thanks

**Comment by gkucmierz** [comment]
Please check this demo

https://gkucmierz.github.io/social-links-app

I belive it does have what you need.

**Comment by bacloud23** [comment]
Exactly. My bad I did not pay attention as part of README I couldn't see it is possible. Thanks a lot !
